### DISCUSSION TITLE
Show if a Review is Resolved

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#4137] [Mobile] [answered]
On iOS app there doesn't seem to be a way to see if a comment is marked as Resolved or not under a PR. Something similar to what is on the web would be quite useful :)

**Comment by aconsuegra** [comment, marked as answer]
Thank you so much for the request! We are tracking being able to show comments as resolved internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

**Reply by risenxxx** [reply]
It would be even better if there will be an ability to mark the conversations as resolved. I mean not only watch, but change the status in both directions, just like in the desktop version.

**Reply by risenxxx** [reply]
There was a mention in the another discussion that most likely this will be released in the end of the month. That's a good news =)

**Comment by ModProg** [comment]
This is very much needed. In the screenshot below you can see 3 Comment threads, the outer two sre resolved but the middle one is qn active discussion. Currently you have to open every winge comment to chech wich ones are already resolved.

![Screenshot_20220301-225653_GitHub.jpg](https://user-images.githubusercontent.com/11978847/156256158-fcbb3c94-2257-447a-b0a6-e793ab345b93.jpg)
